### DISCUSSION TITLE
What's the best way to create two views of the same data?

### DISCUSSION THREAD

**Discussion by EdsterG** [JuliaPlots/Makie.jl#4233] [Q&A]
Suppose I have a slider over a variable that'll generate hundreds of random trajectories:
<img width="320" alt="Screenshot 2023-02-26 at 12 56 44 AM" src="https://user-images.githubusercontent.com/3106867/221401251-4ed8925e-b9cc-48a9-ada6-4ec730568e08.png">

I'd like to create two plots of this data. One of them is a global overview and the other a zoomed in version that's controlled by another slider. Where the second slider will pan along the x-axis to give me a zoomed in view.

Option 1) create two plots of this data and change the limits on the second plot. This seems inefficient because two different plots needs to update hundreds of trajectories.

Option 2) create one plot of the data, and then another plots that's nothing more than a zoomed in "view" of the first plot. This would be more efficient because two different sources don't need to redraw the same data.

Is it even possible to accomplish the second option?

**Comment by EdsterG** [comment]
Forgot to mention this is using `stairs!`.

**Comment by ffreyer** [comment]
Just for people that find this - there isn't an option to reuse a plot so just do 1).
